src/tests/test_blindsig.py

```diff
@@ -1,52 +1,66 @@
-"""
-Test for ECC blind signatures
-"""
-import os
 import unittest
 from ctypes import cast, c_char_p
-
-from pybitmessage.pyelliptic.eccblind import ECCBlind
-from pybitmessage.pyelliptic.openssl import OpenSSL
-
+import sys
+sys.path.insert(1, '/home/cis/ektarepo/PyBitmessage/src/pyelliptic')
+#from eccblind import ECCBlind
+from eccblindChain import ECCBlindChain
+from openssl import OpenSSL
+from datetime import datetime
+from datetime import timedelta
+exp = datetime.now()+(timedelta(days =365))
+signer_obj_list = []
+msg_unpack_Chain = []
 
 class TestBlindSig(unittest.TestCase):
     """
-    Test case for ECC blind signature
+        Class to test blind signature mechanism
     """
     def test_blind_sig(self):
-        """Test full sequence using a random certifier key and a random message"""
-        # See page 127 of the paper
-        # (1) Initialization
-        signer_obj = ECCBlind()
-        point_r = signer_obj.signer_init()
-
-        # (2) Request
-        requester_obj = ECCBlind(pubkey=signer_obj.pubkey)
-        # only 64 byte messages are planned to be used in Bitmessage
-        msg = os.urandom(64)
-        msg_blinded = requester_obj.create_signing_request(point_r, msg)
-
-        # check
-        msg_blinded_str = OpenSSL.malloc(0, OpenSSL.BN_num_bytes(msg_blinded))
-        OpenSSL.BN_bn2bin(msg_blinded, msg_blinded_str)
-        self.assertNotEqual(msg, cast(msg_blinded_str, c_char_p).value)
+        """
+        Method to test blind signature API
+        """
+        level = 0
+        while level <= 4:
+            signer_obj = ECCBlindChain()
```

Comment:
just `ECCBlind`. `ECCBlindChain` should only be used once, on serialising/deserialising/verification.